web/packages/teleport/src/config.ts

```diff
@@ -558,12 +558,16 @@ const cfg = {
     return generatePath(cfg.routes.audit, { clusterId });
   },
 
-  getIntegrationEnrollRoute(type?: string) {
-    return generatePath(cfg.routes.integrationEnroll, { type });
+  getIntegrationEnrollRoute(type?: string, page?: string) {
+    return generatePath(cfg.routes.integrationEnroll, { type, page });
   },
 
-  getIntegrationStatusRoute(type: PluginKind | IntegrationKind, name: string) {
-    return generatePath(cfg.routes.integrationStatus, { type, name });
+  getIntegrationStatusRoute(
+    type: PluginKind | IntegrationKind,
+    name: string,
+    page?: string
```

Comment:
nit: from the PR description, I think we are referring to status page here correct?
```suggestion
    statusPage?: string
```

Reply:
It's for the Edit routes for the User Sync / App/Group Sync within the Okta status page... would something like `subPage` be more apt?

Reply:
`subPage` reads more clear in that case. Add a comment too explaining what that field refers to.